_docs/install/cloud/azure.md

```diff
@@ -5,4 +5,204 @@ anchor: install
+
+Example definition (saved as `kubernetes.json` for use in the instructions below):
+
+```json
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.10"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 3,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    }
+  }
+}
+```
+
+To deploy a cluster through acs-engine you need your Azure Subscription ID. 
+The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) is a great way to get the ID of your subscription. If you don't have the Azure CLI installed then you can sign in to [Azure Cloud Shell](https://shell.azure.com) to run the commands there:
+
+```bash
+# save the current subscription id in a variable for later...
+subscriptionId=$(az account show --output tsv --query id)
+
+# or show the current account (to copy locally if running from the Cloud Shell)
+az account show --output json
+
+# or list your subscriptions (if you have multiple subscriptions)
+az account list
+```
+
+With your subscription id you can now run `acs-engine` to deploy your kubernetes cluster:
+
+```bash
+# IMPORTANT: enter your own parameters, especially 'subscription-id'
+acs-engine deploy --subscription-id $subscriptionId \
+    --dns-prefix storageos --location westus2 \
+    --auto-suffix --api-model kubernetes.json
+```
+
+The deployment also creates a KUBECONFIG for your cluster under the `_output` directory. The exact filename depends on the dns prefix and location you specified. To set the KUBECONFIG for the example above:
+
+```bash
+KUBECONFIG=~/mydir/_output/storageos/kubeconfig/kubeconfig.westus2.json
+```
+
+Running `kubectl cluster-version` should now connect to your new cluster!
+
+Consult the [acs-engine documentation](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes/deploy.md)
+for more information.
+
+### StorageOS installation
+
+First, make sure the [StorageOS CLI]({%link _docs/reference/cli/index.md %}) has
+been installed on your local machine.
+
+Once the Kubernetes cluster is available, use [Helm](https://helm.sh/) to
+install StorageOS.
+
+Once Helm is installed and configured for the cluster, checkout the StorageOS
+Helm Chart and install:
+
+```bash
+git clone https://github.com/storageos/helm-chart.git storageos
+cd storageos
+helm install . --name storageos-test \
+  --set image.repository=soegarots/node \
+  --set image.tag=ad3f0ccb6 \
```

Comment:
@oicheryl /@croomes - should we be specifying a specific tag here?

Reply:
Good catch - 0.10.0 should be the tag.